builder/ebs/builder_acc_test.go

```diff
@@ -76,6 +76,73 @@ func TestAccBuilder_EbsRegionCopy(t *testing.T) {
 	acctest.TestPlugin(t, testCase)
 }
 
+func TestAccBuilder_EbsRegionsCopyWithDeprecation(t *testing.T) {
```

Comment:
> This failed however with encrypted AMIs as encrypting the image after depreciation failed to propagate the attribute to the encrypted image.

Given this statement would it makes sense to add an acceptance test with encryption enabled to ensure that the copy and deprecation attribute are set as expected?

Reply:
The previous PR from @wampiedriessen added such a test to the suite, I can consider adding a new one that ensures the combination of copy + encrypt + depreciate does work, I'll add this to the PR before we can merge it

Reply:
Test for encryption + copy + depreciate added in the latest reroll